<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestureParameterClass</class>
 <widget class="QDialog" name="GestureParameterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GestureParameter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="detectionGroup">
     <property name="title">
      <string>Parámetros de detección</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSpinBox" name="bufferSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="relaxedCount"/>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="distanceCalculatorAGM">
        <item>
         <property name="text">
          <string>Distancia Euclídea</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="nonVaryingAccelerationRate"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="accelerationThreshold"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gestureDescriptionGroup">
     <property name="title">
      <string>Parámetros del gesto</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QSpinBox" name="walkedPixels">
        <property name="minimum">
         <number>15</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="minimunPointCount">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="distanceThreshold">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="maxGestureTime">
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dtwGroup">
     <property name="title">
      <string>Parámetros del reconocimiento</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QComboBox" name="valuesNormalizator">
        <item>
         <property name="text">
          <string>Normalizador Simple</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="distanceCalculatorDTW">
        <item>
         <property name="text">
          <string>Distancia Euclídea</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="relaxedRatio">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="dtwAlgorithm">
        <item>
         <property name="text">
          <string>DTW Simple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paralelogramo de Itakura</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Banda de Sakoe/Chiba</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="rate">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="accept">
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reject">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>bufferSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GestureParameterClass</receiver>
   <slot>setDetectedCount(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>relaxedCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>GestureParameterClass</receiver>
   <slot>setMaxRelatedRatio(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accept</sender>
   <signal>clicked()</signal>
   <receiver>GestureParameterClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reject</sender>
   <signal>clicked()</signal>
   <receiver>GestureParameterClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setDetectedCount(int)</slot>
  <slot>setMaxRelatedRatio(int)</slot>
 </slots>
</ui>
